.husky/pre-commit

```diff
@@ -0,0 +1 @@
+npx lint-staged
```

Comment:
is there a way to add the `generate-graphql-ts-types` from `front/package.json`?

Reply:
I do think so, just need to add the command to the `lint-staged` section in the root `package.json`.